testplan/parser.py

```diff
@@ -152,6 +153,22 @@ def generate_parser(self) -> HelpParser:
             '{"reason": <value>, "strict": <value>} }',
         )
 
+        general_group.add_argument(
+            "--runtime-data",
+            metavar="PATH",
+            type=_runtime_json_file,
+            help="Read a historical runtime data from a JSON file used to auto-part task scheduling "
```

Comment:
"Historical runtime data which will be used for Multitest auto-part and weight-based Task smart-scheduling."